content/memory_model.tex

```diff
@@ -26,12 +26,18 @@
 data objects are symmetric:
 %
 \begin{itemize}
-  \item \Fortran data objects in common blocks or with the  SAVE  attribute.
-      These data objects must not be defined in a dynamic shared object (DSO).
-  \item Global and static \Cstd and \Cpp variables. These data objects must
-      not  be defined in a DSO.
-  \item \Fortran arrays allocated with \textit{shpalloc} 
-  \item \Cstd and \Cpp data allocated by \textit{shmem\_malloc}
+\item
+  \begin{deprecate}
+    \Fortran data objects in common blocks or with the  SAVE  attribute.
```

Comment:
I realize that this line wasn't edited by this proposal.  But, as part of this change, can we format ```SAVE``` using either the ```\TYPE{}``` or ```\TYPENAME{}``` macro?